packages/utils/src/SDKErrors.ts

```diff
@@ -6,348 +6,220 @@
  */
 
 /**
- * SDKErrors are KILT-specific errors, with associated codes and descriptions.
+ * KILT-specific errors with descriptions.
  *
  * @packageDocumentation
  */
 
 /* eslint-disable max-classes-per-file */
 
-export abstract class SDKError extends Error {
-  constructor(...args: ConstructorParameters<ErrorConstructor>) {
-    super(...args)
+export class SDKError extends Error {
+  constructor(message?: string, options?: ErrorOptions) {
+    super(message, options)
+    // this line is the only reason for using SDKError
```

Comment:
that and you are able to identify errors thrown by us with `instanceof SDKError`